src/store/actions.js

```diff
@@ -217,13 +217,14 @@ function fetchLatestVersion({ commit }) {
     });
 }
 
-async function fetchMarkets({ commit }, { done }) {
```

Comment:
line 21 and 29 of this file has formOrder exported twice; for some reason github won't let me comment on the line though. I think it was introduced in the merge though